examples/showcase/post-processing/index.html

```diff
@@ -0,0 +1,65 @@
+<html>
+
+<head>    
+     <script type="importmap">
+        {
+          "imports": {
+            "aframe": "../../dist/aframe-master.module.min.js",
+            "three": "../../../super-three-package/build/three.module.js",
+            "three/addons/": "../../../super-three-package/examples/jsm/",
```

Comment:
How do you test the example? Do you run it with the `npm start` or `npm run start:https` to test with the headset? The example is not running with the trailing comma here.